the-collab-lab/tcl-69-smart-shopping-list
# [KD] Add screenshots of app on About page

## Description

This code replaces the temp phone images on the About page with screenshots of the application to help the user learn how to use the app.

## Related Issue

Closes #91

## Acceptance Criteria

- [x] Swap out the temporary blank phone images with images of screenshots of the app

## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.

## Updates

### Before

![before](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/122697843/a3798f88-1ac6-466c-bc92-7b274393629b)


### After

![after](https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/122697843/df5ffdde-0f9a-4138-9e09-88805258984e)


## Testing Steps / QA Criteria

Navigate to the About page. Scroll down to see that all temp phone images have been replaced with screenshots from the application that compliment the instructions.
